modules/project-docs/pages/compatibility.adoc

```diff
@@ -80,6 +79,16 @@ At time of writing (July 2021): 10.14 (_Mojave_), 10.15 (_Catalina_), and 11 (_B
 
 Although installable or compilable on many other platforms, we cannot provide support for untested combinations.
 
+[NOTE]
```

Comment:
this NOTE could be moved up to the Linux section, as the content only applies to that.

Reply:
linked with anchor tag on note